AutomatedTesting/Gem/PythonTests/prefab/Prefab_Test_Utils.py

```diff
@@ -0,0 +1,103 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+
+from azlmbr.entity import EntityId
+from azlmbr.math import Vector3
+from editor_python_test_tools.utils import Report
+from editor_python_test_tools.utils import TestHelper as helper
+
+import azlmbr.bus as bus
+import azlmbr.components as components
+import azlmbr.editor as editor
+import azlmbr.entity as entity
+import azlmbr.legacy.general as general
+
+
+class UtilsResults:
+    unique_name_entity_found = (
+        "Entity with a unique name found",
+        "Entity with a unique name *not* found")
+
+    prefab_at_expected_position = (
+        "prefab is at expected position",
+        "prefab is *not* at expected position")
+
+    entity_children_count_matched = (
+        "Entity with a unique name found",
+        "Entity with a unique name *not* found")
```

Comment:
same comment on the alignment

Reply:
I don't think that's a good idea since every time when you try to add new results it will have to adjust almost all the previous ones. Also, not all the tests have this kind of convention right now.

Reply:
If you check the physics or the converted largeworlds test for example, they have that.
For avoiding auto-formatting even, they do #fmt-off aswell.

It's quite unlikely that new results will be added in the future as tests thought, I think its worth to improve readability

Reply:
Right now I just declare the tuples inside the function that uses them.